src/spec/index/subscribe.spec.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2017, FinancialForce.com, inc
+ * Copyright (c) 2017-2018 FinancialForce.com, inc
```

Comment:
I believe the policy is to not go copyright hunting:
https://sites.google.com/a/financialforce.com/development-standards/c/ap/copyright-notices-in-source-code-files

> please do not bulk edit existing files to add copyright notices, or edit a file simply to add a copyright notice.